lib/mongo/operation/shared/executable.rb

```diff
@@ -54,7 +54,7 @@ def get_result(server, client)
       def dispatch_message(server, client)
         server.with_connection do |connection|
           message = build_message(server)
-          message = message.maybe_encrypt(client)
+          message = message.maybe_encrypt(client, server)
```

Comment:
Can the argument order be flipped here to match `dispatch_message` signature?

Reply:
Yup!